src/override/malloc.cc

```diff
@@ -18,7 +18,7 @@ extern "C"
 {
   SNMALLOC_EXPORT void* SNMALLOC_NAME_MANGLE(__malloc_end_pointer)(void* ptr)
   {
-    return Alloc::external_pointer<OnePastEnd>(ptr);
+    return ThreadAlloc::get()->external_pointer<OnePastEnd>(ptr);
```

Comment:
This is on the fast path for some things that we have, which makes this look like it's a potentially expensive change.  Why is it needed?  A number of functions were made non-`static` above, but I'm not completely sure that I understand why.

Reply:
I think that commit predates https://github.com/microsoft/snmalloc/pull/86 and is perhaps stale, sorry.  Now that `pagemap()` is static, there's no risk of not finding the external pagemap.  I'll remove the commit from the next `push -f`.